tests/coverage/reachable/assert-false/expected

```diff
@@ -0,0 +1,8 @@
+coverage/reachable/assert-false/main.rs, 6, FULL
+coverage/reachable/assert-false/main.rs, 7, FULL
+coverage/reachable/assert-false/main.rs, 11, FULL
+coverage/reachable/assert-false/main.rs, 12, FULL
+coverage/reachable/assert-false/main.rs, 15, FULL
+coverage/reachable/assert-false/main.rs, 16, FULL
+coverage/reachable/assert-false/main.rs, 17, PARTIAL
```

Comment:
Why is this PARTIAL?

Reply:
Because we override and expand these `assert`s as:
```
    ($cond:expr, $($arg:tt)+) => {{
        kani::assert(!!$cond, concat!(stringify!($($arg)+)));
        // [...]
        if false {
            __kani__workaround_core_assert!(true, $($arg)+);
        }
    }};
```
The code there includes some checks that won't be covered.